indico/modules/events/abstracts/controllers/management.py

```diff
@@ -92,19 +93,43 @@ def _checkParams(self, params):
 
 class RHManageAbstract(RHManageAbstractBase):
     """Display abstract management page"""
+
     def _process(self):
         form = AbstractJudgmentForm(abstract=self.abstract)
         if form.process_ajax():
             return form.ajax_response
-        if form.validate_on_submit():
-            abstract_data = form.data
-            judgment = abstract_data.pop('judgment')
-            send_notification = abstract_data.pop('send_notifications')
-            contrib_session = abstract_data.pop('contrib_session', None)
-            merge_persons = abstract_data.pop('merge_persons', None)
-            judge_abstract(self.abstract, abstract_data, judgment=judgment, contrib_session=contrib_session,
-                           merge_persons=merge_persons, send_notification=send_notification, )
-        return WPManageAbstracts.render_template('abstract.html', self._conf, abstract=self.abstract, form=form)
+        elif form.validate_on_submit():
+            special_data, remaining_data = form.split_data
```

Comment:
These are not very descriptive names. Why is the data special? What is the remaining data?